erpnext/accounts/doctype/opening_invoice_creation_tool/opening_invoice_creation_tool.py

```diff
@@ -32,8 +32,10 @@ def prepare_invoice_summary(doctype, invoices):
 				})
 				invoices_summary.update({company: _summary})
 
-				paid_amount.append(invoice.paid_amount)
-				outstanding_amount.append(invoice.outstanding_amount)
+				if invoice.paid_amount:
```

Comment:
passing invoice.paid_amount and invoice.outstanding_amount inside flt while appending should solve this issue.

Reply:
Multiple ways to solve a problem! This one just seems a much more cheaper way to do it though